### PR TITLE
Commit Cargo.lock

### PR DESCRIPTION
**Pull request by tomhoule**
As suggested in the issue. Maybe we should also check in CI that the Cargo.lock is up to date?

closes https://github.com/pksunkara/pgx_ulid/issues/33

### CODE REVIEW

**Comment by pksunkara** [comment]
yes, please add a job in `ci.yml`

```yaml
  lockfile:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Install rust
        uses: dtolnay/rust-toolchain@1.73.0
      - name: Lockfile check
        run: cargo update -w --locked
```

**Comment by tomhoule** [comment]
Done :+1:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -46,3 +46,13 @@ jobs:
...
           command: fmt
           args: -- --check
+
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install rust
+        uses: dtolnay/rust-toolchain@1.73.0
```
Comment (pksunkara) [inline .github/workflows/ci.yml:56]: This needs to be 1.70.0

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -46,3 +46,13 @@ jobs:
         with:
           command: fmt
           args: -- --check
+
+  lockfile:
```
Comment (pksunkara) [inline .github/workflows/ci.yml:50]: This needs to be added in ci.needs

**Comment by tomhoule** [comment]
Just pushed, both comments should be addressed.

**Comment by pksunkara** [comment]
Thanks for the contribution!

**Pull request merged**